### ISSUE TITLE
Fail to aks update command(with --uptime-sla options)

### ISSUE THREAD

**Issue by ysyukr** [Azure/azure-cli-extensions#3248]
## Describe the bug

**Command Name**
`az aks update
Extension Name: aks-preview. Version: 0.5.7.`

**Errors:**
```
'NoneType' object has no attribute 'get'
Traceback (most recent call last):
python3.8/site-packages/knack/cli.py, ln 233, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 660, in execute
    raise ex
cli/core/commands/__init__.py, ln 723, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
...
    raise value
cli/core/commands/__init__.py, ln 694, in _run_job
    result = cmd_copy(params)
cli/core/commands/__init__.py, ln 331, in __call__
    return self.handler(*args, **kwargs)
azure/cli/core/__init__.py, ln 811, in default_command_handler
    return op(**command_args)
cliextensions/aks-preview/azext_aks_preview/custom.py, ln 1739, in aks_update
    azure_keyvault_secrets_provider_addon_profile = instance.addon_profiles.get(
AttributeError: 'NoneType' object has no attribute 'get'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks update --name {} --resource-group {} --uptime-sla`

## Environment Summary
```
macOS-11.2.3-x86_64-i386-64bit
Python 3.8.9
Installer: HOMEBREW

azure-cli 2.21.0

Extensions:
aks-preview 0.5.7
interactive 0.4.4

```
## Additional Context

I command `az aks update --update-sla` failed.
This seems to be an aks-preview issue.

**Comment by yungezz** [comment]
https://github.com/Azure/azure-cli/issues/14287

**Comment by ysyukr** [comment]
This issue has been confirmed to be resolved through the #3251

**Comment by RakeshMohanMSFT** [comment]
@ysyukr
